app/models/manageiq/providers/vmware/cloud_manager/refresh_parser.rb

```diff
@@ -20,6 +20,8 @@ def ems_inv_to_hashes
     get_vdcs
     get_vapps
     get_vms
+    get_private_images if @options.get_private_images
```

Comment:
But what exactly is image from VMware and MiQ perspective? vmdk disk? And what is VM? It should be something like flavor ...